NGitLab.Mock/Config/GitLabHelpers.cs

```diff
@@ -1398,6 +1456,24 @@ private static Permission CreatePermission(GitLabServer server, GitLabPermission
                 : new Permission(GetOrCreateUser(server, permission.User), permission.Level);
         }
 
+        private static void CreateMilestone(Group group, GitLabMilestone milestone)
+        {
+            var mlt = new Milestone
```

Comment:
Could the creation of the milestone itself be added in a method? To avoid the exact same code in CreateMilestone for group and project.

ex: 
```csharp
private static void CreateMilestone(Group group, GitLabMilestone milestone)
{
  var mlt = CreateMilestone(milestone);
  group.Milestones.Add(mlt);
}

private static void CreateMilestone(Project project, GitLabMilestone milestone)
{
  var mlt = CreateMilestone(milestone);
  project.Milestones.Add(mlt);
}

private static Milestone CreateMilestone(GitLabMilestone milestone)
{
  // Initialization
  // && date logic for closedAt & updatedAt
}
```